.github/workflows/docs-ci.yml

```diff
@@ -50,7 +57,7 @@ jobs:
         if: ${{ github.ref == 'refs/heads/main' }}
         uses: actions/upload-artifact@v3
         with:
-          name: javadocs
+          name: javadocs-${{ steps.dhc-version.outputs.version }}
```

Comment:
Separately... we should probably use this technique for our release artifacts too - much nicer w/ them having version attached...

Reply:
Do you mean like _artifacts-0.30.1_?  Or are you suggesting producing separate artifacts for what that contains?

Reply:
Yeah, just making it so when the server does the publish workflow, the artifacts that it produces has unique names

Reply:
Added a ticket for this: https://github.com/deephaven/deephaven-core/issues/4869.  Feel free to change if it's not what you are thinking.